client/src/components/Home/HomeGridCurrentGlmLock/ModalLockGlm/LockGlmNotification/LockGlmNotification.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useMemo } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import BoxRounded from 'components/ui/BoxRounded';
+import Button from 'components/ui/Button';
+import Svg from 'components/ui/Svg';
+import networkConfig from 'constants/networkConfig';
+import { arrowTopRight, checkMark, notificationIconWarning } from 'svg/misc';
+
+import styles from './LockGlmNotification.module.scss';
+import LockGlmNotificationProps from './types';
+
+const ButtonLinkWithIcon: FC<{ children?: React.ReactNode; transactionHash: string }> = ({
```

Comment:
We gotta extrat these types into `{...}Props` later, right?

Reply:
Good catch. I extracted this.